internal/cmd/db_destroy.go

```diff
@@ -27,31 +28,44 @@ var destroyCmd = &cobra.Command{
 			return err
 		}
 
-		db, err := getDatabase(client, args[0])
-		if err != nil {
-			return nil
-		}
+		if instanceFlag != "" || locationFlag != "" {
```

Comment:
These flags are only supposed to be used on databases that were created before database groups existed. In those databases, instance names are unique - so if you're specifying an instance you shouldn't use more than one DB name.
So if either flags are set I think you should error out if `len(args) > 1`.

Reply:
Thank you! I wasn't sure what to do on this case.
Will make the changes and push.